pkg/coremodel/dashboard/addenda.go

```diff
@@ -0,0 +1,10 @@
+package dashboard
+
+// HandoffSchemaVersion is the minimum schemaVersion for dashboards at which the
+// Thema-based dashboard schema is known to be valid.
```

Comment:
🤣 

```suggestion
// Thema-based dashboard schema is possibly valid
```

we are still gettting training wheels here!